win32ss/user/user32/windows/cursoricon.c

```diff
@@ -152,6 +152,9 @@ static BOOL is_dib_monochrome( const BITMAPINFO* info )
     }
 }
 
+/* Return the size of the bitmap info structure including color table and
+ * the bytes required for 3 DWORDS if this is a BI_BITFIELDS(3) bmp.
+ */
```

Comment:
```suggestion
 * the bytes required for 3 DWORDS if this is a BI_BITFIELDS(3) bmp. */
```
Also, not sure it's useful to mention the value of the BI_BITFIELDS define.

Reply:
Changed and removed the "(3)" for the BI_BITFIELDS define value. Thanks.